Content.Client/Ghost/UI/GhostTargetWindow.xaml.cs

```diff
@@ -28,58 +26,35 @@ public GhostTargetWindow(IEntityNetworkManager netManager)
             _netManager = netManager;
         }
 
-        public void Populate()
+        public void UpdateWarps(IEnumerable<GhostWarp> warps)
         {
-            ButtonContainer.DisposeAllChildren();
-            AddButtonPlayers();
-            AddButtonLocations();
+            _warps = warps.Select(w =>
+            {
+                var name = w.Localizable
+                    ? Loc.GetString("ghost-target-window-current-button", ("name", w.DisplayName))
+                    : w.DisplayName;
+
+                return (name, w.Entity);
+            }).ToList();
+            
+            // Server COULD send these sorted but how about we just use the client to do it instead
+            _warps.Sort((x, y) =>
+                string.Compare(x.Item1, y.Item1, StringComparison.Ordinal));
```

Comment:
Unlike before, some players may now get listed after warp points, which might cause confusion. Players & locations should stay separated

Reply:
Fixed